entry/src/entry.rs

```diff
@@ -111,15 +111,22 @@ pub struct Api<'a> {
 /// field points to Transactions that took place shortly before `hash` was generated.
 ///
 /// If you multiply `num_hashes` by the amount of time it takes to generate a new hash, you
-/// get a duration estimate since the last Entry. Since processing power increases
+/// get a duration estimate since the last `Entry`. Since processing power increases
 /// over time, one should expect the duration `num_hashes` represents to decrease proportionally.
 /// An upper bound on Duration can be estimated by assuming each hash was generated by the
 /// world's fastest processor at the time the entry was recorded. Or said another way, it
 /// is physically not possible for a shorter duration to have occurred if one assumes the
 /// hash was computed by the world's fastest processor at that time. The hash chain is both
 /// a Verifiable Delay Function (VDF) and a Proof of Work (not to be confused with Proof of
 /// Work consensus!)
-
+///
+/// `Entry` contains `transactions` that are executable in parallel.  So an `Entry` is a unit of
+/// parallelism at the chain level.  All transactions in the `transactions` field have to follow the
+/// read/write locking restrictions with regard to the accounts they reference.  An account can be
+/// either written by a single transaction, or read by one or more transactions in the set.
```

Comment:
```suggestion
/// either written by a single transaction, or read by one or more transactions in the entry.
```

Reply:
The way I understand it, two transaction in the same block can still write into the same account, as long as they are not in the same `Entry`.

Reply:
> The way I understand it, two transaction in the same block can still write into the same account, as long as they are not in the same Entry

That is correct. But we're talking about a single entry at this point of the comment, right?
> all transactions in the `transactions` ..

My suggestion to use `entry` over `set` was to reinforce that we're still talking about an `Entry`, given that `set` is a term for another data structure. What do you think about the following instead (ie put "within an entry" at the start over "in the set"  at the end, where "set" is referring to `transactions` from the previous sentence?

"Within an entry, an account can either be written by a single transaction, or read by one or more transactions."

Reply:
Sorry, my response was mixed up.
I must have been thinking about your second comment when writing a response to this one.
I've used `set` to reduce repetition, but it does make it a bit more ambiguous.
I've removed `in the set`.  I think, one can understand that we are talking about `transactions` in the `Entry`.